takwimu/templates/takwimu/_includes/navbar/about.html

```diff
@@ -4,69 +4,71 @@
-            class="col partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
-          <img src="{% static 'img/logos/ap-white.png' %}" class="img-responsive" alt="africapractice">
-        </a>
-        <a href="https://codeforafrica.org" target="_blank" rel="noopener"
-            class="col partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
-          <img src="{% static 'img/logos/cfafrica-white.png' %}" class="img-responsive" alt="Code for Africa">
-        </a>
-        <a href="https://ihub.co.ke/" target="_blank" rel="noopener"
-            class="col partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
-          <img src="{% static 'img/logos/ihub-white.png' %}" class="img-responsive" alt="iHub">
-        </a>
+      <div class="col-12 mt-5">
+        <h5 class="mb-0 text-truncate font-weight-bold">An initiative by </h5>
+        <div class="row">
+          <a href="http://www.africapractice.com/" target="_blank" rel="noopener"
+              class="col partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
+            <img src="{% static 'img/logos/ap-footer-white.png' %}" alt="africapractice">
+          </a>
+          <a href="https://codeforafrica.org" target="_blank" rel="noopener"
+              class="col partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
+            <img src="{% static 'img/logos/cfafrica-white.png' %}" alt="Code for Africa">
+          </a>
+          <a href="https://ihub.co.ke/" target="_blank" rel="noopener"
+              class="col partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
+            <img src="{% static 'img/logos/ihub-white.png' %}" alt="iHub">
+          </a>
+          <div class="col-4">
+          <p class="mb-0 pt-0 text-truncate text-center" style="opacity: 0.6; font-size: 16px;margin-top:-1.5rem;">Supported by:</p>
+            <a href="https://www.gatesfoundation.org" target="_blank" rel="noopener"
+              class="col ml-4 partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
+              <img src="{% static 'img/logos/bmgf-white.png' %}" alt="Bill and Melinda Gates Foundation">
+            </a>
+          </div>
       </div>
     </div>
-    <div class="col-4">
-      <p class="mb-0 text-truncate text-center" style="opacity: 0.6; font-size: 16px">Sponsored by:</p>
-      <a href="https://www.gatesfoundation.org" target="_blank" rel="noopener"
-          class="col-8  ml-4 partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
-        <img src="{% static 'img/logos/bmgf-white.png' %}" class="img-responsive" alt="Bill and Melinda Gates Foundation">
-      </a>
-    </div>
-  </div>
 </div>
+
 <style>
-ul li u {
-  color:#7ab1f2;
-}
+  .border-right.m-2.float-right{
+    height:24px; width:1px; opacity:0.2;
+  }
```

Comment:
Please chose a better selector(s) for this e.g `.navbar .social-icons`. Note: please don't copy-paste my example. Use it as an example and critically think what a selector should be.

Reply:
This selector still needs improvement because:

1. They're chained so wouldn't work i.e would have to be applied in the same `class` attribute to work e.g  
```<div class="navbar-nav nav-item border-right">```  
While the border-right class is more likely in a div child of the navbar.

2. `navbar-nav` is a common class to bootstrap so you're changing all border-right implementation of all the navbar-nav we could have.